### ISSUE TITLE
[FEATURE] Checkout and Opt-In in demo-store

### ISSUE THREAD

**Issue by BrocksiNet** [shopware/frontends#1451]
### Description

Could you test the Checkout process with Opt-In activated and compare it with the Default Storefront Workflow?

### Use Case

When Opt-In is activated, and I try to checkout, I should not be blocked by the Opt-In option for the first checkout.

**Comment by BrocksiNet** [comment]
Duplicate of #1462
